pkg/api-server/testdata/resources/inspect/dataplanes/_rules/resource_rule_meshtimeout_index.golden.json

```diff
@@ -46,17 +46,17 @@
    "toRules": [
     {
      "conf": {
-      "connectionTimeout": "7s",
-      "idleTimeout": "7s",
+      "connectionTimeout": "5s",
```

Comment:
this one is also weird, it's a new `resourceRule` test, so shouldn't be affected no?

Reply:
It's a `toRules` section changing so doesn't affect `resourceRule`